Source/Configuration.py

```diff
@@ -0,0 +1,80 @@
+from Parser.ParserHelpers import getSingleString
+from Parser.PyParser import parseFile, registerKeyword
+import os
+import sys
+
+
+def instantiate(unused, theStream):
+    def verifyImperatorPath(path):
+        if not os.path.isdir(path):
+            sys.exit(path + ' does not exist!')
+        if not os.path.isfile(path + r'\binaries\imperator.exe'):
+            sys.exit(path + ' does not contain Imperator: Rome!')
+        print('\tImperator: Rome install path is ' + path)
+
+    ImperatordirectoryOutput = []
+
+    def getImperatordirectory(unused, theStream):
+        ImperatordirectoryOutput.append('')  # ImperatordirectoryOutput[0] empty, keyword is not used
```

Comment:
It'll be a common enough idiom that if you don't use the first, there's no need to comment on it.